ptbr/tests/test_config_cli.py

```
@@ -0,0 +1,727 @@
+        assert template_path.exists(), f"template.yaml not found at {template_path}"
+
+    def test_template_parses(self, template_path):
+        with open(template_path) as f:
+            data = yaml.safe_load(f)
+        assert isinstance(data, dict)
+        assert "gliner_config" in data
+
+    def test_template_validates_span(self, template_path):
+        result = load_and_validate_config(template_path, full_or_lora="full", method="span")
+        assert result.report.is_valid, (
+            f"Template failed validation:\n"
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)